analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -51,6 +51,8 @@ def lac_gpc_plots(fitspath, fitspath_ini, dataset, raw=False,
 
     gpc_pdf_file = join(fitspath, f"GPC{suffix}.pdf")
     lac_pdf_file = join(fitspath, f"LAC{suffix}.pdf")
+    R23_pdf_diff = join(fitspath, f"LAC{suffix}_diff.pdf")
+    O32_pdf_diff = join(fitspath, f"LAC{suffix}_diff.pdf")
```

Comment:
Please follow the convention and suffix filenames with "_file" much like the two previous lines